hazelcast/src/main/java/com/hazelcast/internal/networking/ChannelWriter.java

```diff
@@ -1,96 +0,0 @@
-/*
```

Comment:
Implementation detail; should not be exposed as a general API. All communication of higher level systems like Connections go through Channel.